Spatial_Engine/Query/ElementCurves.cs

```diff
@@ -33,11 +33,25 @@ namespace BH.Engine.Spatial
 {
     public static partial class Query
     {
+        /******************************************/
+        /****            IElement0D            ****/
+        /******************************************/
+
+        [Description("Queries the defining curves of the IElement0D. Always returns empty collection due to zero-dimensionality of an IElement0D.")]
+        [Input("element0D", "The IElement0D to extract the defining curves from.")]
+        [Input("recursive", "Has no effect for IElement0D. Input here to unify inputs between all IElements.")]
+        [Output("elementCurves", "The curves defining the base geometry of the IElement0D.")]
```

Comment:
```suggestion
        [Description("Queries the defining curves of an IElement0D. Always returns empty collection due to zero-dimensionality of an IElement0D.")]
        [Input("element0D", "The IElement0D to extract the defining curves from.")]
        [Input("recursive", "Has no effect for IElement0D. Input here to unify inputs between all IElements.")]
        [Output("elementCurves", "The curves defining the base geometry of the IElement0D.")]
```
Same as above, I guess? Maybe I'm to picky at this point.. 🙈